virkarekisteri/virkarekisteri
# VIR-47 Fetching data to table from the backend

Added position-slice where getPositions and addPosition hooks can be called
 Fetches data to the table

How to test:
1. Open virkarekisteri
2. Check that it fetches data from backend to the table
3. Test adding new position and see that it is appended to the table
